tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -774,8 +774,17 @@ def setUpClass(cls):
         cls.loc_name = cls.loc.name
         cls.config_env = configure_provisioning(compute=True,
                                                 org=cls.org,
-                                                loc=cls.loc
+                                                loc=cls.loc,
+                                                os=cls.rhev_img_os
                                                 )
+        cls.os_name = cls.config_env['os']
+        subnet = entities.Subnet().search(
```

Comment:
`config_env` already has information about subnet or not?

Reply:
yes it does , I am just updating the subnet that we got from config_env , it is also used by discovery tests , so didn't want to change anything in the helper itself